src/components/Navigation/NavigationItems/NavigationItems.js

```diff
@@ -5,9 +5,9 @@ const navigationItems = (props) => {
 
     return (
             <ul className= {classes.NavigationItems}> 
-                  <NavigationItem Link="/"  excat >Add To Do</NavigationItem>
-                  <NavigationItem Link="/Charts"  excat >Charts</NavigationItem>
-                  <NavigationItem Link="/Auth"  excat >Autenticate</NavigationItem>
+                  <NavigationItem Link="/" exact >Add To Do</NavigationItem>
+                  <NavigationItem Link="/chart" >Charts</NavigationItem>
+                  <NavigationItem Link="/Auth" >Autenticate</NavigationItem>
```

Comment:
Using the constants from a file that will be shared between Link and Routes. So we can avoid any typos and it will allow us to change routes text in one place instead of going over different places

Reply:
Done